mkdocs.yml

```diff
@@ -24,9 +24,13 @@ theme:
     accent: 'deep blue'
   feature:
     tabs: false
+
 extra_css:
   - 'assets/style/theme.css'
 
+markdown_extensions:
+    - attr_list
```

Comment:
what does that do? :)

Reply:
It allows to resize the image inside the md file. Like so:
`![img-resources-example-2.png](assets/images/img-resources-example-2.png){ width=50% }`